eq-author-api/migrations/updateMetadataValue.test.js

```diff
@@ -0,0 +1,73 @@
+const updateMetadataValue = require("./updateMetadataValue.js");
+
+describe("updateMetadataValue", () => {
+  it("should update metadata to correct new format", () => {
+    const metadata = [
+      {
+        alias: "text",
+        dateValue: null,
+        id: "296",
+        key: "ru_ref",
+        languageValue: null,
+        regionValue: null,
+        textValue: "old Value",
+        type: "Text",
+        value: "new Value",
+      },
+      {
+        alias: "date",
+        dateValue: "2019-07-01",
+        id: "294",
+        key: "date",
+        languageValue: null,
+        regionValue: null,
+        textValue: null,
+        type: "Date",
+        value: "2019-04-09",
+      },
+      {
+        alias: "region",
+        dateValue: null,
+        id: "297",
+        key: "region",
+        languageValue: null,
+        regionValue: "GB_GBN",
+        textValue: null,
+        type: "Region",
+        value: "GB_ENG",
+      },
+      {
+        alias: "language",
+        dateValue: null,
+        id: "297",
+        key: "language",
+        languageValue: "cy",
+        regionValue: null,
+        textValue: null,
+        type: "Language",
+        value: "en",
+      },
+      {
+        alias: "language",
+        dateValue: null,
+        id: "297",
+        key: "language",
+        languageValue: "cy",
+        regionValue: null,
+        textValue: null,
+        type: "Region",
```

Comment:
Are you testing a specific scenario in this last assertion that I'm missing? It feels like this in last entry is invalid configuration. In theory you should never be able to specify a `Region` metadata with a non-null `languageValue`?